tests/pytests/integration/ssh/test_state.py

```diff
@@ -0,0 +1,282 @@
+import pathlib
+import shutil
+import threading
+import time
+
+import pytest
+
+from tests.support.runtests import RUNTIME_VARS
+
+pytestmark = [
+    pytest.mark.slow_test,
+    pytest.mark.skip_on_windows(reason="salt-ssh not available on Windows"),
+]
+
+SSH_SLS = "ssh_state_tests"
+SSH_SLS_ID = "ssh-file-test"
+SSH_SLS_FILE = pathlib.Path("/tmp/salt_test_file")
```

Comment:
Don't we want to avoid creating global level variables? Shouldn't these be fixtures instead?